src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -287,12 +281,14 @@ def _performChecks(self):
             self.parameters["LoadAverage"] = []
         self.parameters["LoadAverage"].append(loadAvg)
 
-        memoryUsed = self.getMemoryUsed()
-        msg += f"MemUsed: {memoryUsed:.1f} kb "
-        heartBeatDict["MemoryUsed"] = memoryUsed
+        result = self.profiler.memoryUsage(withChildren=True)
+        if not result["OK"]:
+            self.log.warn("Could not get rss info from profiler", result["Message"])
+        msg += f"MemUsed: {result['Value']:.1f} mb "
```

Comment:
Is there any reason to quote "mb" units in lower case?

Reply:
Correcting.